### PR TITLE
i18n Instance for Registration

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes 5139 .

#### Changes proposed in this Pull Request:

- Add i18nRegistrationInstance
- Add shared and Registration dictionary 
- Add Demo screen to test translations

#### Screenshots / Screen Recording (if applicable)

- App Level Translation
Screenshot 2024-02-02 at 3 49 01 PM

- Demo screen from auth workflow package

Screenshot 2024-02-02 at 3 52 21 PM

Screenshot 2024-02-02 at 3 55 15 PM

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Comment by JeffGreiner-eaton** [comment]
I'm able to use the translations for each example and works

image 
image

should the intellisense be working for when we type REGISTRATION. (dot) to invoke like the STEPS or INSTRUCTIONS. (dot) then display the options? I can't remember is this was working before

File: login-workflow/docs/language-support.md [hunk 1]
```diff
@@ -1,56 +1,64 @@
...
 -   French
 -   Portuguese
 -   Spanish
 -   Simplified Chinese
 
-## Changing the Language
+## Add Custom Language
 
-When the app is loaded, you will need to set the language that will be used by the `i18next` object via the `changeLanguage` function:
+If you would like to support a language other than the ones supported by the workflow, you can, but you will need to provide the translations for all of the strings that are needed by the workflow screens. If you do not, the workflow screens will default to showing English.
```
Comment (JeffGreiner-eaton) [inline login-workflow/docs/language-support.md:13]: This section could live in the customization.md similar to the updated docs in react-workflow

Comment (JeffGreiner-eaton) [reply login-workflow/docs/language-support.md:13]: similar to this
https://github.com/etn-ccis/blui-react-workflows/blob/dev/login-workflow/docs/language-support.md

**Review by surajeaton — APPROVED** [review]
Works fine. 
Please remove the demo screen before merging.
Also update the documents suggested by Jeff.

Simulator Screenshot - iphone 12 pro - 2024-02-06 at 12 57 22

Simulator Screenshot - iphone 12 pro - 2024-02-06 at 12 58 05

Simulator Screenshot - iphone 12 pro - 2024-02-06 at 12 58 32

**Comment by ektaghag-eaton** [comment]
> Works fine. Please remove the demo screen before merging. Also update the documents suggested by Jeff.
> 
> Simulator Screenshot - iphone 12 pro - 2024-02-06 at 12 57 22
> 
> Simulator Screenshot - iphone 12 pro - 2024-02-06 at 12 58 05
> 
> Simulator Screenshot - iphone 12 pro - 2024-02-06 at 12 58 32

Hi @surajeaton , will keep the demo screen till the cleanup story (remove demo screens). Added a comment to remove the test code.

**Pull request merged**